### ISSUE TITLE
A11y_Azure Data Studio_Publish Changes_Preview Database Updates_Keyboard: The user is unable to access the tooltip info "Publish Changes" associated with 'Publish' button using keyboard tab keys unlike mouse.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/azuredatastudio#19566]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev3;#SH_AzureDataStudioforMAC_May2022;#SQL Azure Data Studio;#MAC;#FTP;#DesktopApp;#WCAG2.1.1;#Keyboard;#AINotInScope;#LinkedBug;#Linked:Bug132344;

## Environment Details:
Application Name: Azure Data Studio
Application Version: 1.36.1
Mac Version: macOS Monterey 12.4

## Repro Steps:
1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch SQL Azure Data Studio and connect to server.
3. Tab to 'SERVERS' section and expand.
4. Tab till 'Databases' menu item and expand.
5. Select any of the databases and expand.
6. Tab till 'Tables' sub-menu item.
7. Right click on table sub-menu item and enter 'New Table'.
8. Tab till 'Publish Changes' button just above the 'Table name' edit field.
9. Use keyboard to access the tooltip info.

## Actual:
The user is unable to access the tooltip info "Publish Changes" associated with 'Publish' button using keyboard tab keys unlike mouse.

## Expected:
The user should be able to access the tooltip info "Publish Changes" associated with 'Publish' button using keyboard tab keys.

## User Impact:
Keyboard users will lose out important tooltip information since there is no visible label is present for the abovementioned control and won't be able to locate the button.

## Attachment:
![8_Tooltip info](https://user-images.githubusercontent.com/98393966/170928264-88f501b6-9a47-4a94-a519-d4e92bac4416.gif)

**Comment by alanrenmsft** [comment]
@bbarik06 this is an issue we inherited from vscode: https://github.com/microsoft/vscode/issues/132344

as per the external bug process, I am closing the issue.

**Comment by bbarik06** [comment]
@alanrenmsft Bugs can be only closed by trusted testers as per the external bug process it does say that the primary bug should be in resolved state. Since, we don't have any resolved state in Github, the bug should remain in Open state. We have reduced the severity to sev3 of this bug as per the guidance from the external bug. So this bug will not affect your grading. Please reopen this bug.

**Comment by Charles-Gagnon** [comment]
@Jtith I'm confused at the requirement here. Is there actually an expectation that tooltips show up when you tab into a button? That'd get pretty annoying having that pop up constantly and is not something I've seen in other apps. Can you link to the WCAG item for this? (or other documented requirement)

**Comment by Jtith** [comment]
@Charles-Gagnon You can refer MAS reference link [MAS 2.1.1 – Keyboard](https://aka.ms/MAS2.1.1) for better understanding of keyboard and tooltip related issue also this bug would be closed once issue is fixed throughout the application as we are closing the Duplicate bug https://github.com/microsoft/azuredatastudio/issues/20876

**Comment by fsteffi** [comment]
**GithubTags**:#External;#External:VSCode;
